games/minetest/Portfile

```diff
@@ -44,27 +45,44 @@ post-extract {
     }
 }
 
-license                 LGPL-2.1+
+license                 LGPL
```

Comment:
revert to original as that is what upstream lists in the readme.md file

Reply:
OK

I am not to lecture anybody and I may have been wrong with LGPL and LGPL-2.1 anyway.

See: https://www.thehyve.nl/articles/open-source-software-licenses-part-3